certified-connectors/jbhunt/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2809 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "JBHunt",
+		"description": "Access to committed, market-relevant rating, tendering, and tracking. Connected to nearly 1M trucks through carriers on 360 Marketplace by J.B. Hunt.",
+		"version": "1.0",
+		"contact": {
+			"name": "Microsoft Dynamics",
+			"url": "https://aka.ms/iomsupport",
+			"email": "iomdri@microsoft.com"
+		}
```

Comment:
@vechall Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.